src/main/drivers/bus_i2c.h

```diff
@@ -37,7 +38,163 @@ typedef enum I2CDevice {
     I2CDEV_COUNT
 } I2CDevice;
 
+#ifdef UNIT_TEST
+# define I2C_TypeDef unsigned long
+#endif
```

Comment:
Rather than define `I2C_TypeDef` here under `UNIT_TEST` flag, in `src/test/unit/platform.h` declare:
```
typedef struct
{
    void* test;
} I2C_TypeDef;
```

This fits in with style of other structs defined for `UNIT_TEST`